kubernetes/resource_kubernetes_namespace_v1.go

```diff
@@ -39,7 +40,7 @@ func resourceKubernetesNamespaceV1() *schema.Resource {
 			},
 		},
 		Timeouts: &schema.ResourceTimeout{
-			Delete: schema.DefaultTimeout(5 * time.Minute),
+			Delete: schema.DefaultTimeout(2 * time.Minute),
```

Comment:
I think you used this one to validate doc-gen, could you please revert this change or share more details? :)